src/store/movie/movie.actions.ts

```diff
@@ -0,0 +1,46 @@
+import {createAction, props} from '@ngrx/store';
+import {Movie, SORT} from '@model/movies.model';
+
+export const deleteAllMovies = createAction(
+  '[MOVIE] DELETE_ALL_MOVIES'
+);
+
+export const addToFavourite = createAction(
+  '[MOVIE] ADD_TO_FAVOURITE',
+  props<{ id: string }>()
+);
+
+export const deleteFromFavourite = createAction(
+  '[MOVIE] DELETE_FROM_FAVOURITE',
+  props<{ id: string }>()
```

Comment:
tutaj możesz zrobić taki myk: `id: Movie['id']` - wtedy jak zmieni Ci się typ id w appce to będziesz miał to ograne z automatu